protocol/daemons/liquidation/client/sub_task_runner.go

```diff
@@ -50,6 +52,16 @@ func (s *SubTaskRunnerImpl) RunLiquidationDaemonTaskLoop(
 		return err
 	}
 
+	if lastCommittedBlockHeight == s.lastLoopBlockHeight {
+		daemonClient.logger.Info(
+			"Skipping liquidation daemon task loop as no new block has been committed",
+			"blockHeight", lastCommittedBlockHeight,
+		)
+		return nil
+	}
+
+	s.lastLoopBlockHeight = lastCommittedBlockHeight
```

Comment:
Nit: I wonder if here's the best place to assign `s.lastLoopBlockHeight = lastCommittedBlockHeight` since we haven't sent any current block info to application yet. Wdyt think about deferring the statement `s.lastLoopBlockHeight = lastCommittedBlockHeight`?

Reply:
good question. thought about it a little bit more 

putting it here strictly guarantees that loop doesn't run for the same block height, regardless of errors. If we defer the statement to the end (e.g. after `SendLiquidatableSubaccountIds`), theoretically it's possible that `SendLiquidatableSubaccountIds` only partially succeeds when first few requests goes through and later requests fail. In this case, we would be running into the same issue. I think might be slightly better here for correctness?

Reply:
`defer`ing guarantees that the `s.lastLoopBlockHeight = lastCommittedBlockHeight` is always run before return, even if `SendLiquidatableSubaccountIds` fails right? Trying to understand when we'd send `SendLiquidatableSubaccountIds`again for the same block. 

No strong preference though, the current implementation also works. Up to you. Approved to unblock

Reply:
ohh you mean `defer` and update block height regardless of error? I misunderstood

yeah I think it's basically the same - let's keep it as is to unstuck the testnet first